Setting Classifier does not work

I'm trying to set the Classifier, but I'm getting an error 400.

```log

==============================================================================
Task         : Upload BOM to Dependency Track
Description  : Upload a BOM file to Dependency Track platform.
Version      : 1.14.0
Author       : Edouard Shaar
Help         : 
==============================================================================
Reading BOM from location: C:\agent1\_work\20\a\sboms\sbom.json
Starting BOM upload to Dependency Track at url: http://dtrack:8091/
Project with name 'TEST' and version '5.0.0' will be created if it does not exist.
Getting project id using name 'TEST and version '5.0.0'.
Uploading the BOM succeeded! Token: 210157e0-24b7-406c-bb38-98c2108b7e06
Current project settings.
Id: cb522562-3be3-4dd5-a88e-ebf0f5c5c772
Name: TEST
Version: 5.0.0
Description: undefined
Classifier: undefined
SWID Tag ID: undefined
Group: undefined
Tags: []
Is Latest: false
Updating project.
Finished task execution with error - Error: Failed to update project: 400 - Bad Request
    at DtrackManager.updateProject (C:\agent1\_work\_tasks\upload-bom-dtrack_151c89e2-4c73-4d0f-b81b-bbbd418542ba\1.14.0\dtrackManager.js:86:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async run (C:\agent1\_work\_tasks\upload-bom-dtrack_151c89e2-4c73-4d0f-b81b-bbbd418542ba\1.14.0\task.js:61:3)
##[error]Failed to update project: 400 - Bad Request

```

This is my pipeline:
```yaml
...
variables:
...
  Version: ''  # Populated from GitVersion
  projectName: 'TEST'    
  dtrackServiceConnection: 'DependencyTrack'
  dtrackProjectName: $(projectName)
  dtrackProjectVersion: '$(Version)'
  dtrackProjClassifier: 'Application'

...

- task: upload-bom-dtrack@1
  displayName: 'Upload SBOM to Dependency-Track'
  inputs:
    serviceConnection: '$(dtrackServiceConnection)'
    dtrackProjVersion: '$(dtrackProjectVersion)'
    dtrackProjName : '$(dtrackProjectName)'
    bomFilePath: '$(Build.ArtifactStagingDirectory)/sboms/sbom.json'
    dtrackProjClassifier: '$(dtrackProjClassifier)'    
    dtrackProjAutoCreate: true    
```

If I remove  ``    dtrackProjClassifier: '$(dtrackProjClassifier)'    ``, then It works

--- a/README.md
+++ b/README.md
@@ -91,7 +91,7 @@ Provide **one** of the following:
 | `dtrackProjTags` | Set the prohject tags. (Each tag on a new line) |
 | `dtrackProjSwidTagId` | Set the project SWID Tag Id |
 | `dtrackProjGroup` | Set the project Namespace / group / vendor identifier |
-| `dtrackProjClassifier` | Classifier (e.g., `APPLICATION`, `FRAMEWORK`, etc.) |
+| `dtrackProjClassifier` | Classifier for the project (e.g., `APPLICATION`, `FRAMEWORK`, etc.). Must be uppercase when used in YAML pipelines without quotes. |
 | `dtrackParentProjName` | Parent project name (with optional auto-create) |
 | `dtrackParentProjVersion` | Parent project version (with optional auto-create) |
 | `dtrackIsLatest` | Sets the project as the latest version. Defaults to false. |

